src/openforms/submissions/migrations/0067_alter_submission_language_code.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.16 on 2022-11-07 14:27
+
+import django.utils.translation
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("submissions", "0066_remove_submissionvaluevariable_language"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="submission",
+            name="language_code",
+            field=models.CharField(
+                choices=[("nl", "Dutch"), ("en", "English")],
+                default=django.utils.translation.get_language,
```

Comment:
I think this migration is needed in addition to `0065` to ensure that existing submissions have their language_code set to `""`

Reply:
exactly my train of thoughts :+1: